lib/error/include/hse/error/merr.h

```diff
@@ -105,7 +100,7 @@ struct merr_info {
 /* Not a public API, called only via the merr() macro.
  */
 merr_t
-merr_pack(int error, enum hse_err_ctx ctx, const char *file, const int line);
+merr_pack(int error, int16_t ctx, const char *file, const int line);
```

Comment:
I guess the problem here is that it can only accomodate a small number of user-supllied error codes...
I don't really care for calling it a "context", maybe I'll think of something later...